sentry_sdk/scrubber.py

```diff
@@ -59,19 +59,36 @@
 
 
 class EventScrubber(object):
-    def __init__(self, denylist=None):
-        # type: (Optional[List[str]]) -> None
+    def __init__(self, denylist=None, recursive=False):
+        # type: (Optional[List[str]], bool) -> None
         self.denylist = DEFAULT_DENYLIST if denylist is None else denylist
         self.denylist = [x.lower() for x in self.denylist]
+        self.recursive = recursive
+
+    def scrub_list(self, lst):
```

Comment:
This should likely be marked as private API. Since we have already merged this PR, I have opened #2769 to implement this change – we need to merge #2769 before a release containing this PR is released since it would otherwise be a breaking change.

Reply:
Never mind, following discussion with @sentrivana we have agreed that this can stay public for consistency with the rest of the class